instances/unitOfMeasurement/kilobyte.jsonld

```diff
@@ -10,5 +10,5 @@
   "knowledgeSpaceLink": null,
   "name": "kilobyte",
   "preferredOntologyIdentifier": null,
-  "synonym": null
+  "synonym": ["KB"]
```

Comment:
should be kB ([Wikipedia](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&cad=rja&uact=8&ved=2ahUKEwiCqO66p_nzAhWwxIUKHbSRAoIQmhN6BAhBEAI&url=https%3A%2F%2Fen.wikipedia.org%2Fwiki%2FKilobyte&usg=AOvVaw11-EA9rHiNzVp3r_LYsIhC))